### PR TITLE
Add placeholders for ctl and status

### PR DESCRIPTION
**Pull request by rminnich**
ctl will be used instead of writing strings via write(),
which showed the value of control strings (to me) but
but the implementation is a kludge.

Further, we need to have build time checking of the strings,
and better rules around how it works.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
gonna try this again based on nick's suggestions; I have a lot to learn about Rust. But the idea of breaking out ctl and stat is still something I want to do.

File: src/lib/consts/src/lib.rs [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+// Since it is not exclusive to UARTs we have it here.
+pub enum Baud {
+    B115200
+}
+
+// DeviceCtl is the (small!) 
+pub enum DeviceCtl {
+    On,
+    Off,
+    TTY{Baud: Baud}
```
[the remark this answers is not in the document]
Comment (rminnich) [reply src/lib/consts/src/lib.rs]: done

File: src/arch/x86/x86_64/Cargo.toml [hunk 2]
```diff
@@ -3,8 +3,10 @@ name = "arch"
 version = "0.1.0"
 authors = ["Oreboot Authors"]
 edition = "2018"
+consts = { path = "../../../lib/consts"}
```
Comment (orangecms) [inline src/arch/x86/x86_64/Cargo.toml:6]: That shouldn't be here... or should it? I wonder why CI wouldn't complain. Maybe anything can be put here without restrictive validation.

**Review by orangecms — CHANGES_REQUESTED** [review]
looks okay except for that tiny thing in the x86_64/Cargo.toml

**Review by orangecms — APPROVED** [review]

**Pull request merged**